### PR TITLE
Increase code coverage to 100%

### PR DESCRIPTION
**Pull request by snmgian**

### CODE REVIEW

File: spec/lib/ruy/conditions/any_spec.rb [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
+      expect(condition.call(context)).to be
+    end
+
+    it 'is true when some other condition is met' do
+      context = Ruy::Context.new({:enabled => true})
+
+      expect(condition.call(context)).to be
+    end
+
+    it 'is false when none condition are met' do
```
Comment (migl) [inline spec/lib/ruy/conditions/any_spec.rb]: ..."when no condition is met" or "when no conditions are met".

File: spec/lib/ruy/rule_set_spec.rb [hunk 2]
```diff
@@ -3,6 +3,28 @@
 
 describe Ruy::RuleSet do
 
+  describe '#apply?' do
```
Comment (migl) [inline spec/lib/ruy/rule_set_spec.rb]: Didn't we remove this method?

File: spec/lib/ruy/rule_set_spec.rb [hunk 3]
```diff
@@ -3,6 +3,28 @@
...
+      set
+    end
+
+    it 'is true when conditions met' do
+      rule_set.call({ :flag => true })
+
+      expect(rule_set.apply?).to be
+    end
+
+    it 'is false when conditions do not met' do
```
Comment (migl) [inline spec/lib/ruy/rule_set_spec.rb]: ..."do not meet"

File: spec/lib/ruy/time_pattern_spec.rb [hunk 4]
```diff
@@ -427,4 +427,13 @@
...
   end
 
+  describe 'method delegation' do
+    it 'delegates to its UTC based Time instance' do
+      expect(subject.gmt_offset).to eq(subject.utc.gmt_offset)
+    end
+
+    it 'delegates to super by default' do
+      expect(subject.class).to eq(Ruy::TimePattern)
+    end
```
Comment (migl) [inline spec/lib/ruy/time_pattern_spec.rb]: I'm confused by the test and/or the description. How does this test that methods are delegated to super?

File: spec/lib/ruy/rule_set_spec.rb (hunk 2 above)
Comment (snmgian) [inline spec/lib/ruy/rule_set_spec.rb]: It's still there. There's a backlog task for that.

Comment (snmgian) [inline spec/lib/ruy/rule_set_spec.rb]: Removed

File: spec/lib/ruy/rule_set_spec.rb (hunk 3 above)
Comment (snmgian) [inline spec/lib/ruy/rule_set_spec.rb]: Removed

File: spec/lib/ruy/time_pattern_spec.rb (hunk 4 above)
Comment (snmgian) [inline spec/lib/ruy/time_pattern_spec.rb]: The test was wrong. #class invocation don't fire up method_missing.

File: spec/lib/ruy/conditions/any_spec.rb (hunk 1 above)
Comment (snmgian) [inline spec/lib/ruy/conditions/any_spec.rb]: Done

**Pull request merged**